test/openfisca/mapping/individu-helpers.coffee

```diff
@@ -0,0 +1,44 @@
+individuHelpers = require '../../../lib/simulation/openfisca/mapping/common'
+should = require 'should'
+
+describe 'isIndividuValid', ->
+  SITUATION =
+    dateDeValeur: new Date '2015-01'
+
+  describe 'an adult', ->
+    target =
+      role: 'demandeur'
+      situationsPro: []
+      dateDeNaissance: new Date '1940-01'
+
+    it 'should be valid', ->
+      individuHelpers.isIndividuValid(target, SITUATION).should.be.ok
+
+  describe 'a child', ->
+    target =
+      role: 'enfant'
+
+    describe 'under 25', ->
+      before ->
+          target['situationsPro'] = []
+          target['dateDeNaissance'] = new Date '2010-01'
+
+      it 'should be valid', ->
+        individuHelpers.isIndividuValid(target, SITUATION).should.be.ok
+
+    describe 'over 25', ->
+      before ->
+        target['situationsPro'] = []
```

Comment:
The order of execution of `describe`, `before` and `it` is the order of declaration, so you can build upon the previous assignments and simply set `dateDeNaissance` here (same below).